README.md

```diff
@@ -98,6 +98,22 @@ This project uses [CMake] and [Hunter] package manager.
    sudo make install
    ```
 
+#### OpenCL support on Linux
```

Comment:
It might be a good idea to mention that this is only relevant for hunter. If you disable hunter, there are different requirements I assume.